packages/portfolio-contract/test/portfolio.flows.test.ts

```diff
@@ -1127,18 +1127,19 @@ test('handle failure in provideCosmosAccount makeAccount', async t => {
 
   const attempt1 = rebalance(orch, ctx, seat1, { flow: steps }, kit);
 
-  await t.throwsAsync(
-    attempt1,
-    { message: 'timeout creating ICA' },
-    'rebalance should fail when noble account creation fails',
-  );
+  t.is(await attempt1, undefined);
 
   // Check failure evidence
   const failCall = log.find(entry => entry._method === 'fail');
   t.truthy(
     failCall,
     'seat.fail() should be called when noble account creation fails',
   );
+  t.deepEqual(
+    failCall!.reason,
+    Error('timeout creating ICA'),
+    'rebalance should fail when noble account creation fails',
```

Comment:
As noted elsewhere, this was sort of a proxy for testing that openPortfolio should fail when noble account creation fails.